global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.505"
+    "version": "2.1.802"
```

Comment:
Could you please revert this - this would break our CI. 

I suggest 2 ways to deal with this if this causes trouble during local development:
- The best is to install `2.1.505`, then all project in this repo will be able to pick up that version, all the other projects on your machine will use the newest version. 
- Do this change, but do not commit the `global.json` file.

Reply:
Reverted.